paper-toolbar.html

```diff
@@ -233,150 +233,120 @@
       }
     </style>
 
-    <div id="topBar" class$="[[_computeBarClassName(justify)]]">
+    <div id="topBar" class$="center horizontal layout toolbar-tools [[_computeBarExtraClasses(justify)]]">
       <content select=":not(.middle):not(.bottom)"></content>
     </div>
 
-    <div id="middleBar" class$="[[_computeBarClassName(middleJustify)]]">
+    <div id="middleBar" class$="center horizontal layout toolbar-tools [[_computeBarExtraClasses(middleJustify)]]">
       <content select=".middle"></content>
     </div>
 
-    <div id="bottomBar" class$="[[_computeBarClassName(bottomJustify)]]">
+    <div id="bottomBar" class$="center horizontal layout toolbar-tools [[_computeBarExtraClasses(bottomJustify)]]">
```

Comment:
I'd recommend to create a local class and use mixins instead. These classes are coming from paper-styles and they use `/deep/` which is deprecated.